static/app/bootstrap/initializeSdk.tsx

```diff
@@ -93,6 +93,15 @@ export function initializeSdk(config: Config, {routes}: {routes?: Function} = {}
       }
       return tracesSampleRate;
     },
+    beforeSendTransaction(event) {
+      event.spans = event.spans?.filter(span => {
+        // Filter analytic timeout spans.
+        return ['reload.getsentry.net', 'amplitude.com'].every(
```

Comment:
@k-fish Wondering if we should do it for other things as well like `https://data.pendo.io`

Reply:
We can, for now I'm just going to add these, I'd have to look first and make sure Pendo or other 3rd parties don't have any interaction with our pages (eg. slow resources, pop-ups that have some interaction), whereas I'm more sure of reload/amp are send-only.

Reply:
@k-fish well yea they do have pop-ups that might be slow but I was hoping to ignore them